### PR TITLE
Age range for wisher

### PR DESCRIPTION
**Pull request by amlidos**
### What was the feature/problem?

{There needed to be field checking so that kids outside of the make a wish range can't accidentally create a wish.}

### How does this PR address the above feature/problem?

{This PR adds and addresses all of the functionality requirements in the issue}

### Check lists (check `x` in `[ ]` of list items)

- [x] Test passed
- [x] Coding style (indentation, etc)

### Additional Comments (if any)

{Please write here}

### CODE REVIEW

**Comment by WRMilling** [comment]
Hey @amlidos, thanks for jumping on #74. If you don't mind, could you please add the tests corresponding to the added logic?

**Comment by amlidos** [comment]
> Hey @amlidos, thanks for jumping on #74. If you don't mind, could you please add the tests corresponding to the added logic?

You're welcome, sure I'll add tests in for the logic.

**Comment by amlidos** [comment]
I finished adding in the tests

File: ui/src/landing/CreateWish.test.js [hunk 1]
```diff
@@ -25,28 +25,34 @@ describe('CreateWish tests', () => {
...
+
+    it('Should have validFields return true', () => {
+      const instance = wrapper.instance();
+      const {validAge} = instance.validFields(instance.state.age);
+      expect(validAge).toEqual(true);
+    })
   })
 
-  describe('When any of the states (name, age, and wish type) is absent', () => {
+  describe('When the states (name, age, and wish type) are invalid', () => {
```
Comment (crstamps2) [inline ui/src/landing/CreateWish.test.js]: @amlidos I received a little more information on the age range for make a wish. https://wish.org/content/faq#Who-can-receive-a-wish

can you update the range to be between 2 (inclusively) and 18 (exclusively)

Comment (amlidos) [reply ui/src/landing/CreateWish.test.js]: Sure, I updated the code to check for that, updated the tests, and caught my fork up with the upstream master.

**Pull request merged**